cli/tests/test_release_workflow_staged.py

```
@@ -338,8 +405,23 @@ def test_release_accepts_dispatch_sha_reachable_from_reviewed_main() -> None:
+@pytest.mark.parametrize(
+    ("target", "expected"),
+    (
+        (
+            "0.8.7",
+            ["0.8.6", "0.8.5", "0.8.4", "0.7.2", "0.6.6", "0.5.0"],
+        ),
+        (
+            "0.8.8",
+            ["0.8.7", "0.8.6", "0.8.5", "0.8.4", "0.7.2", "0.6.6", "0.5.0"],
+        ),
+    ),
```

Ruff (0.15.21)

[warning] Wrong values type in `pytest.mark.parametrize` expected `list` of `tuple`

Use `list` of `tuple` for parameter values

(PT007)